lib/vault/encrypted_model.rb

```diff
@@ -297,11 +344,15 @@ def __vault_persist_attribute!(attribute, options)
         # Generate context if needed
         generated_context = __vault_generate_context(context)
 
-        # Generate the ciphertext and store it back as an attribute
-        ciphertext = Vault::Rails.encrypt(
-          path, key, plaintext,
-          context: generated_context
-        )
+        if transform
+          ciphertext = plaintext
+        else
+          # Generate the ciphertext and store it back as an attribute
+          ciphertext = Vault::Rails.encrypt(
```

Comment:
Not sure if we should expand on the API in this case to support this, but Transform uses different terminologies here for these equivalent operations: `encoded_value`, `encode`, and `decoded_value`, `decode`.

Reply:
Looks like we do actually have `transform_decode` and `transform_decode`, just not the field names.

Reply:
Nevermind, it looks like it does properly parse those response values.